### DISCUSSION TITLE
A question on the large eddy simulation model.

### DISCUSSION THREAD

**Discussion by esaka-forever** [xcompact3d/Incompact3d#218] [Q&A] [answered]
Dear  xcompact3d community,
First of all, I am using the 4.0 version (I guess this may be one cause of my issue). I realize when specifying the LES model. some example folders have used "jles = " while some parameter file in the channel folder is using "iles = ". So which one is actually functioning? It looks like parameter "iles"  is not actively used in the program?
Meanwhile, when I use “jles” to simulate a implicit LES (by setting jles=4), the results is exactly the same as when I set jles=0. I remember some xcompact3d paper says the implicit LES has adopted a more dissipative scheme. So how to activate that dissipative scheme? Any help is appreciated !!!

**Comment by slaizet** [comment, marked as answer]
Hi,
`iles` has now been removed in the latest versions of the code. To activate the LES mode, you need to have `ilesmod=1` and then you can select which `jles` you want.
Best,
Sylvain

**Reply by esaka-forever** [reply]
Thank you for the reply. Another issue is that, when I want to use implicit les, I set jles=4, but the results are exactly the same as when I set ilesmod=0. Is that normal? I think the Xcompact3d paper said the implicit les has adopted a more dissipative scheme somewhere.

**Reply by slaizet** [reply]
As per my previous message, you need `ilesmod=1` to activate the LES mode. 
Best,
Sylvain

**Reply by esaka-forever** [reply]
Thank you.

**Comment by brunoscapin1** [comment]
esaka, if you want to use the iles method make sure you also set isecondder = 5, so that the code calculates the scheme's coeff considering the isvv constrains (see schemes.f90 l.467 for more info).

**Reply by esaka-forever** [reply]
I see. Thank you for the help !
